webpack/assets/javascripts/react_app/components/Architectures/Welcome.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../common/I18n';
+import EmptyState from '../common/EmptyState';
+import { foremanUrl } from '../../common/helpers';
+
+export const WelcomeArchitecture = ({ canCreate }) => {
+  const action = canCreate && {
+    title: __('Create Architecture'),
+    url: foremanUrl('/architectures/new'),
+  };
+  const description = (
+    <>
+      <p>
+        {__(
+          'Before you proceed to using Foreman you should provide information about one or more architectures.'
+        )}
+        <br />
+        {__(
+          'Each entries represent a particular hardware architecture, most commonly'
+        )}
+        <b>{__(' x86_64')}</b> {__('or')} <b>{__('i386.')}</b> <br />
```

Comment:
This (and other similar changes) could break the translations in different languages where the sentence structure is different - the translators will not see the full sentence and will not be able to reorder it to match the correct sentence structure in their language.
You would probably need to include the tags inside the translated string and use `dangerouslySetInnerHTML` function to get the translated string converted to HTML.
It would be ideal if you could keep as many of the strings as possible identical to what they were previously, so that existing translations can still be used.